### ISSUE TITLE
variables undefined after task.sleep

### ISSUE THREAD

**Issue by dlashua** [custom-components/pyscript#37]
This issue is dependent on ... something else ... that I can't quite figure out. It doesn't happen every time, so a simplified test doesn't show the issue immediately.

In a Class Method:
```python
        start_sleep = time.monotonic()
        task.sleep(self.timeout)
        elapsed_sleep = time.monotonic() - start_sleep
```

Error:
```
2020-10-11 06:14:12 ERROR (MainThread) [custom_components.pyscript.apps.occupancy_manager.inner_occupied_states] Exception in <apps.occupancy_manager.inner_occupied_states> line 224:
            elapsed_sleep = time.monotonic() - start_sleep
                                               ^
NameError: name 'start_sleep' is not defined
```

**Comment by dlashua** [comment]
This appears to be happening outside of a class as well.

**Comment by craigbarratt** [comment]
So far I'm not able to replicate this.  Could this case be connected to reload?

**Comment by dlashua** [comment]
Perhaps? I just restarted Home Assistant (to add full traceback). I will need to restart pyscript once when it starts (because for some reason my YAML defined apps don't load without a restart, I've yet to debug that and report). But, aside from that one restart, I'll leave it be for a bit and see if it happens again.

**Comment by craigbarratt** [comment]
It looks like the new config flow check breaks the initial `config` settings, which requires a reload.

**Comment by dlashua** [comment]
Ah, glad you found that. I still hadn't gotten around to writing a proper, concise test case for it. Thanks!

**Comment by dlashua** [comment]
The original issue (variables undefined after task.sleep) seems to have been resolved. I will open a new issue if it presents again.

Leaving this open for now in case you're using it to track "failure to load initial `config` values".

**Comment by craigbarratt** [comment]
The config loading is being addressed in #39.
